diff --git a/node_modules/@coinbase/wallet-sdk/dist/sign/walletlink/WalletLinkSigner.js b/node_modules/@coinbase/wallet-sdk/dist/sign/walletlink/WalletLinkSigner.js
index f717750..ee2042a 100644
--- a/node_modules/@coinbase/wallet-sdk/dist/sign/walletlink/WalletLinkSigner.js
+++ b/node_modules/@coinbase/wallet-sdk/dist/sign/walletlink/WalletLinkSigner.js
@@ -1,7 +1,7 @@
 // Copyright (c) 2018-2024 Coinbase, Inc. <https://www.coinbase.com/>
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
-import * as eip712 from '../../vendor-js/eth-eip712-util/index.cjs';
+import eip712 from '../../vendor-js/eth-eip712-util/index.cjs';
 import { LOCAL_STORAGE_ADDRESSES_KEY } from './relay/constants.js';
 import { isErrorResponse } from './relay/type/Web3Response.js';
 import { WalletLinkRelay } from './relay/WalletLinkRelay.js';
